types/npmcli__package-json/tslint.json

```
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
+    }
```

```suggestion
    "extends": "@definitelytyped/dtslint/dt.json"
```
why you would need this rule exclusion? It looks perfect NPM kind of package, resolvable via NPM

---

Without this, I CI wouldn't pass

https://github.com/DefinitelyTyped/DefinitelyTyped/runs/6330895335?check_suite_focus=true

---

it's about the error I've mentoned, this is a CJS module:
```txt
ERROR: 1:1   npm-naming  The declaration doesn't match the JavaScript module 'npmcli__package-json'. Reason:
The declaration should use 'export =' syntax because the JavaScript source uses 'module.exports =' syntax and 'module.exports' can be called or constructed.
```
once fixed, this exclusionn won't be needed I think